contentcuration/contentcuration/static/js/exercise_creation/views.js

```diff
@@ -0,0 +1,562 @@
+var Backbone = require("backbone");
+var _ = require("underscore");
+var ExerciseModels = require("./models");
+var $ = require("jquery");
+var Quill = require("quilljs");
+var Dropzone = require("dropzone");
+var get_cookie = require("utils/get_cookie");
+var UndoManager = require("backbone-undo");
+var JSZip = require("jszip");
+var fileSaver = require("browser-filesaver");
+var JSZipUtils = require("jszip-utils");
+
+require("../../less/exercises.less");
+require("quilljs/dist/quill.snow.css");
+require("dropzone/dist/dropzone.css");
+
+var ExerciseListView = Backbone.View.extend({
+
+    initialize: function() {
+        this.render();
+        this.listenTo(this.collection, "add", this.render);
+        this.listenTo(this.collection, "remove", this.render);
+    },
+
+    events: {
+        "click .create": "add_exercise",
+        "click .delete": "delete_exercise"
+    },
+
+    template: require("./hbtemplates/exercise_list.handlebars"),
+
+    render: function (){
+        this.$el.html(this.template({exercise_list: this.collection.toJSON()}))
+    },
+
+    add_exercise: function() {
+        this.collection.create();
+    },
+
+    delete_exercise: function(ev) {
+        var id = ev.currentTarget.value;
+        var model = this.collection.get(id);
+        model.destroy();
+        return false;
+    }
+});
+
+var FileUploadView = Backbone.View.extend({
+
+    initialize: function(options) {
+        _.bindAll(this, "file_uploaded");
+        this.callback = options.callback;
+        this.modal = options.modal;
+        this.render();
+    },
+
+    template: require("./hbtemplates/file_upload.handlebars"),
+
+    modal_template: require("./hbtemplates/file_upload_modal.handlebars"),
+
+    render: function() {
+
+        if (this.modal) {
```

Comment:
If the behavior begins to diverge too much in future iterations, you might consider making to different View types.